feature: Add information about tag discard without common ancestor (#40)
codacy/pulse-user-docs: docs/one-click-integrations.md

    -   The set of changes that belong to a deployment is the list of commits between the tag of that deployment and the previous tag.

        Because of this, Pulse discards the first SemVer tag in the repository since there is no previous tag to compare with.

>         Also, in case a tag (and its previous tag) do not have a common ancestor, since we cannot obtain their changes, it will be discarded.

Reviewer: ```suggestion
        Pulse also discards any tag that does not have a common ancestor (commit) with its previous tag, since we cannot obtain the changes between them.
```

Reviewer: Actually, in the last part of the sentence, instead of "we" it should be "it".

>     -   The set of changes that belong to a deployment is the list of commits between the tag of that deployment and the previous tag.
> 
>         Because of this, Pulse discards the first SemVer tag in the repository since there is no previous tag to compare with.
>         Pulse also discards any tag that does not have a common ancestor (commit) with its previous tag, since it cannot obtain the changes between them.

Reviewer 2: Suggestion to condense the text a bit, as the description of how Pulse works with Git tags is growing quite large:

```suggestion
    -   The set of changes that belong to a deployment is the list of commits between the tag of that deployment and the previous tag. Because of this, Pulse discards:
        
        -   The first SemVer tag in the repository since there is no previous tag to compare with.
        -   Any tag that does not have a common ancestor (commit) with its previous tag, since Pulse cannot obtain the changes between them.
```
